R/dummy_functions.R

```diff
@@ -62,7 +62,7 @@ example_module <- function(label = "example teal module",
 
         table_data_decorated_no_print <- srv_transform_teal_data(
           "decorate",
-          data = table_data,
+          teal_data_r = table_data,
```

Comment:
I'm torn between `r_teal_data` and `teal_data_r`. Ask yourself how would you read in your mind this code `reactive(teal_data())`? In my mind it's `reactive teal data`. It doesn't matter what we use in `tmg` or `tmc`. It's actually `tmg` and `tmc` that should follow `teal`. Anyway, not a big deal

Reply:
Ok, yes this was perhaps the more controversial decision while renaming, I will rename to r_real_data. I too think it will be easier to read  (and maybe leave a note/issue on tmg and tmc to match this).